keras/regularizers.py

```diff
@@ -16,6 +18,33 @@ def get_config(self):
         return {'name': self.__class__.__name__}
 
 
+class EigenvalueRegularizer(Regularizer):
```

Comment:
A docstring was included in the class EigenvalueRegularizer. I also added a test for the new regularizer in _test_regularizers.py_, here is the output, running on Theano backend in my machine:
======================== test session starts ==================================
platform linux2 -- Python 2.7.6, pytest-2.9.1, py-1.4.31, pluggy-0.3.1
rootdir: /home/oswaldo/Documents/EigenvalueDecay/Eigenvalue_new_version_Keras, inifile: 
collected 3 items 
test_regularizers.py ...
================== 3 passed in 10.78 seconds ==================================
However, I am experiencing an issue to get the shape of a tensor in the class _EigenvalueRegularizer_: the type returned for shape is different for Theano backend (Theano tensor type) and TF backend (TF TensorShape). Maybe we have to revise the files _theano_backend.py_ and _tensorflow_backend.py_
, to uniformize the output?